src/table/cells/handleKeyPress.js

```diff
@@ -0,0 +1,74 @@
+const navigationEffect = (tableRows, nextRow, nextCol) => {
+  const nextCell = tableRows[nextRow].getElementsByClassName('sn-table-cell')[nextCol];
```

Comment:
I know my prototype relied on classnames, but can it be done by using tags instead? `td` for the content and `th` for the head

Reply:
Personally, I think both getElementsByClassName and getElementsByTagName are okay. Do you have any specific reason to prefer tag?

Reply:
Not needing to add extra classes to the components. All classnames from mui are renamed to a unique code, like `1614601815238-sn-t44` (1614601815238 is a unique seed for that chart, sn-t for sn-table, then a number to identify which class it is). Not that pretty combining with something like `sn-table-cel'` since it wont be converted. That being said, I looked around and it seems fine to mix classes like this. I just think we can avoid adding the classes since they just correspond to a tag anyways

Reply:
I do not think adding extra class names affects anything, which means avoiding adding the classes does not get any benefits.  
And adding class names and getting elements by those class names are a kind of common practice. 
I argue that it is compensated conciseness for its readability and understandability.

Reply:
good points, let's stick with classes!